src/test/java/jrm/user/UserSignInTest.java

```diff
@@ -3,13 +3,16 @@
 import io.restassured.http.ContentType;
 import io.restassured.response.ValidatableResponse;
 import jrm.commonclasses.CommonTest;
+import jrm.dao.UserRepository;
 import jrm.dto.UserDto;
 import jrm.service.UserAuthApiService;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.context.annotation.Description;
 
+import java.util.Optional;
```

Comment:
Where this import is used? May be I missed something.

Reply:
We use it in test body!

Spring libraries are used for defining test-methods as beans. 
'UserDto' interface is reloaded to create new 'user'-object, which is converting into JSON and sending to API (singin or signup)
'UserAuthApiServices' is imported as service because eponymous java-class stores end-points values and contain getter for token. 

That's how I understood it, may it's wrong.